js/index.js

```diff
@@ -2,8 +2,13 @@
 
 function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
-
   //... your code goes here
+  const price = product.querySelector('.price span').innerHTML;
```

Comment:
```suggestion
  const price = product.querySelector('.price span').innerText;
```
Siempre que puedas evitarte usar el HTML hazlo!, no dudes :D